cookbooks/vankeken_subduction/vankeken_corner_flow.prm

```diff
@@ -0,0 +1,167 @@
+end
+
+# Use the specialized geometry model for this cookbook
+subsection Geometry model
+  set Model name = vanKeken box
+end
+
+# Checkpoint every 200 steps
+subsection Checkpointing
+  set Steps between checkpoint = 200
+end
+
+# Prescribe a half space cooling model on the left boundary and in the overriding plate
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Variable names = x,y,t
+    # H is the model Y extent, Hl is the overriding plate thickness, TS is the surface temperature
+    # T0 is the temperature at the base of the slab
+    set Function constants = H=6.0e5, Hl=5.0e4, TS=273.0, T0=1573.0, kappa=0.7272e-6, t50=1.5778463e15
+    set Function expression = if(x<=(H-y),TS+(T0-TS)*(1.0-erfc( (H-y)/(2.0*sqrt(kappa*t50))) ), \
+                 if(x>(H-y) & y>(H-Hl), TS+(T0-TS)*(H-y)/Hl, \
+                    T0 ) )
+  end
+end
+
+# Bottom and right boundary is open
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = bottom:zero traction
+end
+
+# Keep the temperature fixed on the left, top and right boundaries
+subsection Boundary temperature model
+  set List of model names                           = initial temperature
+  set Allow fixed temperature on outflow boundaries = false
+  set Fixed temperature boundary indicators         = top, left, right
+end
+
+# Prescribe a velocity of 5 cm/yr on the left boundary, 0 on the top
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = left: function, top:function
+  subsection Function
+    set Variable names = x,y,t
+    set Function constants = H=6.01e5, v=0.035355339
+    set Function expression = if( x<=(H-y), v, 0.0) ; \
+                              if( x<=(H-y),-v, 0.0 )
+  end
+end
+
+# No gravity
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 0.
+  end
+end
+
+# Use the visco plastic material model and specify and isoviscous material with
+# a viscosiy of 1e20 Pa s
+subsection Material model
+  set Model name         = visco plastic
+  set Material averaging = harmonic average only viscosity
+  subsection Visco Plastic
+    set Viscous flow law                          =     diffusion
+    set Viscosity averaging scheme                =     harmonic
+
+    set Stress exponents for diffusion creep      =     1 # No stress dependence
+    set Prefactors for diffusion creep            =     1e-50 # ensure we reach the max viscosity
+    set Activation energies for diffusion creep   =     0 # no temperature dependence
+    set Activation volumes for diffusion creep    =     0 # no pressure dependence
+    set Grain size                                =     1
+    set Grain size exponents for diffusion creep  =     1
+
+    set Densities                                 =     3300
+    set Heat capacities                           =     1250
+    set Thermal conductivities                    =     3.3
```

Comment:
Having both thermal conductivity and diffusivity is redundant, so I suggest remove one. If you want to specify the thermal conductivity, you need to also set the parameter `set Define thermal conductivities = true`.